base_geoengine/static/src/js/widgets/geoengine_widgets.js

```diff
@@ -207,6 +207,9 @@ odoo.define('base_geoengine.geoengine_widgets', function (require) {
             // Default extent
             if (map_view) {
                 var extent = this.defaultExtent.split(', ');
+                Object.keys(extent).forEach(key => {
+                    extent[key] = Number(extent[key])
+                });
```

Comment:
Using `map` might be simpler here.

```suggestion
                extent = extent.map(coord => Number(coord));
```